docs/DeveloperGuide.md

```diff
@@ -262,31 +262,272 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `Pawbook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Add a dog profile**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User request to add a dog profile to the list.
+2.  Pawbook adds the dog.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. Dog name is not specified.
 
-  Use case ends.
+    * 1a1. Pawbook shows an error message.
+    * 1a2. User adds dog name.
```

Comment:
This might sound like a bit of nitpicking, but then if I don't other teams might, the "adds" in this sentence can be confusing, perhaps we can use something like append/include/insert? Same for the things below.